src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -439,8 +439,7 @@ void primitive_inst::set_arguments() {
 }
 
 void primitive_inst::build_deps() {
-    if (_node == nullptr)
-        return;
+    OPENVINO_ASSERT(_node != nullptr, "_node should not be nullptr for build_deps.");
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1021124364
Updated to throw an exception when _node is null. Thank you.